PKGBUILD

```diff
@@ -37,15 +37,36 @@ build() {
   cd "$srcdir/atom-$pkgver"
 
   export PYTHON=python2
-  script/build --build-dir "$srcdir/atom-build"
+  script/build --compress-artifacts
```

Comment:
What's the motivation behind this? At first glance i see no reason to compress the binary, as we are not using the generated tarball.